hack/jenkins/e2e.sh

```diff
@@ -649,10 +649,10 @@ case ${JOB_NAME} in
     : ${E2E_DOWN:="true"}
     : ${E2E_TEST:="false"}
     : ${USE_KUBEMARK:="true"}
-    : ${NUM_MINIONS:="40"}
-    : ${E2E_ZONE:="asia-east1-a"}
-    : ${MASTER_SIZE:="n1-standard-32"}  # Note: not available in all zones
-    : ${MINION_SIZE="n1-standard-16"}   # Note: can fit about 17 hollow nodes per core
+    : ${NUM_MINIONS:="8"}
```

Comment:
To save some 16 core VMs - we need ~ 1000/17 = 59 cores worth of nodes, which is e.g. 8*8, or 4 * 16, but I think that it's easier to get 8 8-core machines, than 4 16-core ones.